multimedia/libvpx-devel/Portfile

```diff
@@ -51,14 +51,15 @@ compiler.blacklist-append  {*gcc-[3-4].*} {clang < 900} {macports-clang-3.*}
 # doesn't install docs or examples correctly, so disable them.
 configure.args      --enable-vp8 \
                     --enable-vp9 \
+                    --enable-vp9-highbitdepth \
                     --enable-internal-stats \
                     --enable-pic \
                     --enable-postproc \
                     --enable-multithread \
                     --enable-experimental \
+                    --enable-examples \
```

Comment:
I'm fine with adding examples to the mix, but you'll need to explicitly destroot them:

```
MacPro2012:work chrisadmin$ gfind libvpx-1.12.0/examples -type f -perm 755 -print | gsort
libvpx-1.12.0/examples/decode_to_md5
libvpx-1.12.0/examples/decode_with_drops
libvpx-1.12.0/examples/postproc
libvpx-1.12.0/examples/set_maps
libvpx-1.12.0/examples/simple_decoder
libvpx-1.12.0/examples/simple_encoder
libvpx-1.12.0/examples/twopass_encoder
libvpx-1.12.0/examples/vp8cx_set_ref
libvpx-1.12.0/examples/vp9_lossless_encoder
libvpx-1.12.0/examples/vp9_spatial_svc_encoder
libvpx-1.12.0/examples/vp9cx_set_ref
libvpx-1.12.0/examples/vpx_temporal_svc_encoder
```

I'd recommend they be placed in directory `${prefix}/share/${my_name}/examples/`.

Reply:
> I'm fine with adding examples to the mix, but you'll need to explicitly destroot them

Unless there's a Makefile option to include them in the installation, that is. Haven't looked too closely, but perhaps there is one...?

Reply:
I only enabled examples so vpxenc/vpxdec executables are built/installed, do you want all of these executables to be installed? All of these need to be fixed with install_name_tool if so.

Reply:
> I only enabled examples so vpxenc/vpxdec executables are built/installed, do you want all of these executables to be installed? All of these need to be fixed with install_name_tool if so.

Gotcha, didn't realize those were part of the examples. Never Mind!  :-D